addons/im_livechat/static/src/core/web/channel_invitation_patch.xml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-inherit="discuss.ChannelInvitation.main" t-inherit-mode="extension">
+        <xpath expr="//*[@name='selectablePartnerDetail']//*" position="inside">
+            <span t-if="props.thread?.channel_type === 'livechat' and selectablePartner.user_livechat_username" class="overflow-hidden opacity-75 smaller mx-2">
+                <span class="text-truncate" t-esc="selectablePartner.user_livechat_username"/>
+            </span>
+        </xpath>
         <xpath expr="//*[@name='selectablePartnerDetail']" position="replace">
             <t t-if="props.thread?.channel_type !== 'livechat' or !selectablePartner.lang_name">$0</t>
-            <div t-else="" class="d-flex flex-column flex-grow-1">
+            <div t-else="" class="d-flex flex-column flex-grow-1 overflow-hidden">
```

Comment:
Do we really need these extra hidden?

Usually if the parent width (in all the chain) is properly restricted, this should not be necessary.

It might be because of `li` in the chain though, which might not be the idea tag to use in this situation.

I don't have a suggestion for a fix, it's not critical if we have too keep the extra class.